experimental/rust-tide/templates/default/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+pub fn app() -> tide::Server<()> {    
+    let mut api = tide::new();
+    api.at("/hello").get(|_| async move { Ok("Hello, world!") });
```

Comment:
Could we change this to be reached at the default endpoint? I.e. just `localhost:8000` - It was a bit confusing for me to see a 404 and then have to go back to the README to discover the endpoint was `/hello` for just a basic message. Maybe I'm just used to the way other stacks do it.